ledger/ledgercore/accountdata.go

```diff
@@ -135,10 +135,15 @@ func (u *AccountData) Suspend() {
 }
 
 // Suspended returns true if the account is suspended (offline with keys)
-func (u *AccountData) Suspended() bool {
+func (u AccountData) Suspended() bool {
```

Comment:
Why is `Suspend` a pointer receiver while `Suspended` and `LastSeen` then?

Reply:
`Suspended` and `LastSeen` are observers, `Suspend` mutates.